src/main/scala/ch/epfl/bluebrain/nexus/kg/search/QueryResultEncoder.scala

```diff
@@ -0,0 +1,48 @@
+package ch.epfl.bluebrain.nexus.kg.search
+
+import cats.syntax.show._
+import ch.epfl.bluebrain.nexus.commons.types.search.QueryResults.{ScoredQueryResults, UnscoredQueryResults}
+import ch.epfl.bluebrain.nexus.kg.config.Contexts
+import ch.epfl.bluebrain.nexus.rdf.Iri.AbsoluteIri
+import io.circe.{Encoder, Json}
+
+object QueryResultEncoder {
+
+  /**
+    * Encoder for scored query results
+    */
+  implicit val scoredEncoder: Encoder[ScoredQueryResults[AbsoluteIri]] = Encoder.instance { results =>
+    Json.obj(
```

Comment:
Same here. use `.addContext(Contexts.searchCtxUri)` and use a graph instead of building a pure json